tests/shared/reducers/message.test.js

```diff
@@ -34,4 +37,34 @@ describe("reducers/app", () => {
       message: null,
     });
   });
+
+  describe("test addErrorToState function", () => {
+    it("addErrorMessage with a valid string message", () => {
+      const state = addErrorToState(initialState, {
+        error: "An error message",
+      });
+
+      expect(state).toEqual({
+        ...initialState,
+        message: "An error message",
+      });
+    });
+
+    it("addErrorMessage with an Error instance", () => {
+      const state = addErrorToState(initialState, {
+        error: Error("An error message"),
+      });
+
+      expect(state).toEqual({
+        ...initialState,
+        message: "An error message",
+      });
+    });
+
+    it("addErrorMessage with a non valid error", () => {
```

Comment:
```
it("ignores invalid error types", () => {
```

this test really makes me think something should be done. we should not silent this behavior, it is definitely too dangerous.

Reply:
I should throw a new error ? Maybe accepting only a string is to restrictive, all "printable" variable should be allowed, I should also test numbers with something linke

`false === ["string", "number"].includes(typeof message)`

WDYT ?

Reply:
mmh I think no one would use a number as an error, we should not enforce that.

and yeah, an error for developers would be nice.